### PR TITLE
SSL on gizra.com

### LINKED ISSUE #593: Add SSL to gizra.com [1h]
And force https. Because why not :)

### PR DESCRIPTION
**Pull request by AronNovak**
## Status

got rid of http:// resources and internal links

## Todo

Cloudflare change

will fix #593

### CODE REVIEW

**Comment by AronNovak** [comment]
I configured CloudFlare, https://www.gizra.com/ goes through there, but it's not enforced, it will be after we deploy this PR.

**Comment by IshaDakota** [comment]
@AronNovak looks good - Is there a reason we shouldn't just make all self site references relative?

**Comment by AronNovak** [comment]
Right, this is the best moment to do that, pushed a new commit.

File: src/_posts/2015-02-01-automatic-qa.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ One time setup:
 git clone git@github.com:Gizra/Gizra.git
 cd Gizra
 cd behat
-# By default the base URL is set to http://gizra.com
+# By default the base URL is set to
```
Comment (IshaDakota) [inline src/_posts/2015-02-01-automatic-qa.md]: I think this needs to remain.

File: src/contact/index.html [hunk 2]
```diff
@@ -39,7 +39,7 @@
 
       <button class="ui primary button" type="submit">Send</button>
 
-      <input type="hidden" name="_next" value="http://gizra.com/contact/thank-you">
```
Comment (IshaDakota) [inline src/contact/index.html]: this also because it's a hidden value sent to an external form collector.

Comment (AronNovak) [inline src/contact/index.html]: Indeed, restored.

File: src/_posts/2015-02-01-automatic-qa.md (hunk 1 above)
Comment (AronNovak) [inline src/_posts/2015-02-01-automatic-qa.md]: Ah, for sure :)

**Comment by IshaDakota** [comment]
merged and deployed.

**Pull request merged**